en/docs/learn/creating-geo-location-dataset.md

```diff
@@ -0,0 +1,43 @@
+# Creating Geo Location Data Set
+
+Follow the procedure below in order to create a Geo Location dataset to use when [Enabling Geolocation Based Statistics
+](../../learn/enabling-geolocation-based-statistics).
+
+1.  Create an account in [www.maxmind.com](https://www.maxmind.com/) and download the **GeoLite2 City: CSV Format** as a ZIP file.
+2.  Download the geoip-2-csv-converter from <https://github.com/maxmind/geoip2-csv-converter/releases> according to your operating system.
+
+#### Prepare the database entries
+
+1.  Unzip the latest CSV file and the geoip-2-csv-converter you have downloaded in the steps above.
+2.  Run `update-geolocation-data.sh` file using the command below.
+
+    For Linux  : [update-geolocation-data.sh](../../assets/attachments/learn/geo-location/linux/update-geolocation-data.sh)
+    
+    For Mac: [update-geolocation-data.sh](../../assets/attachments/learn/geo-location/mac/update-geolocation-data.sh)
+
+    ```shell
+    sh update-geolocation-data.sh
+    ```      
+
+    -   Enter the path to the extracted GeoLite2-City-Blocks-IPv4 directory which you downloaded first, as the response for **Enter path to GeoLite2-City-Blocks-IPv4 directory:**
```

Comment:
Please make this a part of the numbered list.

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2716/commits/005760e83eea6c18412d91e41b204fc75bf066fa